backend/supplements/getCharacterDetails_LOL.js

```diff
@@ -0,0 +1,68 @@
+const request = require('request-promise');
+const cheerio = require('cheerio');
+const { PrismaClient } = require('@prisma/client');
+const prisma = new PrismaClient();
+
+async function getCharacterDetails_LOL() {
+  const url = 'https://www.leagueofgraphs.com/champions/builds';
+  try {
+    const html = await request({
+      uri: url,
+      headers: {
+        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3',
+        'Accept': 'text/html',
+        'Accept-Language': 'en-US'
+      }
+    });
+
+    const $ = cheerio.load(html);
+    const characterStats = [];
+
+    $('tr').each((index, element) => {
+      const character = $(element).find('td').eq(1).find('.name').text().trim();
+      const pickrate = parseFloat($(element).find('td').eq(2).find('progressBar').attr('data-value')) * 100;
+      const winrate = parseFloat($(element).find('td').eq(3).find('progressBar').attr('data-value')) * 100;
+      const banrate = parseFloat($(element).find('td').eq(4).find('progressBar').attr('data-value')) * 100;
```

Comment:
from a math point of view I would keep this as decimals between 0 and 1 and only multiply by 100 when you are doing the display because calculations are easier to do with these values instead of percents